test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -385,13 +389,14 @@ public void It_implicitly_defines_compilation_constants_for_the_target_framework
             };
 
             getValuesCommand
-                .Execute()
+                .Execute(addDefineFromCli ? ["/p:DefineConstants=HELLOWORLD"] : [])
                 .Should()
                 .Pass();
 
             var definedConstants = getValuesCommand.GetValues();
+            var expectedConstants = expectedDefines.Concat(addDefineFromCli ? ["HELLOWORLD"] : ["DEBUG", "TRACE"]);
```

Comment:
Not sure if skipping `TRACE` and the configuration define is the best possible behavior, but I'm not going to change it in this PR. We can either open an issue about it, or issue guidance to discourage people from setting `DefineConstants` from the command line.